client/pages/Login.tsx

```diff
@@ -0,0 +1,130 @@
+import React, {useState, useEffect} from 'react';
+import { KeyboardAvoidingView, StyleSheet, Text, TextInput, TouchableOpacity, View } from 'react-native';
+import { useTheme } from '@react-navigation/native';
+import { useNavigation } from '@react-navigation/core';
+
+import { firebase, auth, provider, signInWithGoogle } from '../configs/firebase';
+
+const styles = StyleSheet.create({
+    container: {
+      flex: 1,
+      justifyContent: 'center',
+      alignItems: 'center',
+    },
+    inputContainer: {
+      width: '80%'
+    },
+    input: {
+      backgroundColor: 'white',
+      paddingHorizontal: 15,
+      paddingVertical: 10,
+      borderRadius: 10,
+      marginTop: 5,
+    },
+    buttonContainer: {
+      width: '60%',
+      justifyContent: 'center',
+      alignItems: 'center',
+      marginTop: 40,
+    },
+    button: {
+      backgroundColor: '#0782F9',
+      width: '100%',
+      padding: 15,
+      borderRadius: 10,
+      alignItems: 'center',
+    },
+    buttonOutline: {
+      backgroundColor: 'white',
+      marginTop: 5,
+      borderColor: '#0782F9',
+      borderWidth: 2,
+    },
+    buttonText: {
+      color: 'white',
+      fontWeight: '700',
+      fontSize: 16,
+    },
+    buttonOutlineText: {
+      color: '#0782F9',
+      fontWeight: '700',
+      fontSize: 16,
+    },
+});
+
+const Login = ({navigation}) => {
+    const [email, setEmail] = useState('');
+    const [password, setPassword] = useState('');
+
+    // const navigation = useNavigation()
```

Comment:
The '[navigation'](https://reactnavigation.org/docs/navigation-prop) prop is passed automatically to each Screen element by way of the Stack.Navigator in App.tsx, so this line would be redundant.